leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -40,22 +40,30 @@
  */
 public class RedisSecurityStore implements EditableSecurityStore {
 
-    private static final String SEC_EP = "SEC#EP#";
+    private final String secEp;
 
-    private static final String PSKID_SEC = "PSKID#SEC";
+    private final String pskIdSec;
 
     private final Pool<Jedis> pool;
 
     private final List<SecurityStoreListener> listeners = new CopyOnWriteArrayList<>();
 
     public RedisSecurityStore(Pool<Jedis> pool) {
         this.pool = pool;
+        this.secEp = "SEC#EP#";
+        this.pskIdSec = "PSKID#SEC";
+    }
```

Comment:
Probably better to use : 

```suggestion
    public RedisSecurityStore(Pool<Jedis> pool) {
        this(pool, "SEC#EP#", "PSKID#SEC");
    }
```

Reply:
Sure, thank you.